hassio/src/supervisor-base-element.ts

```diff
@@ -36,18 +41,24 @@ declare global {
   }
 }
 
+const empty = () => "";
+
 export class SupervisorBaseElement extends urlSyncMixin(
   ProvideHassLitMixin(LitElement)
 ) {
   @property({ attribute: false }) public supervisor?: Supervisor;
 
+  @internalProperty() private _localize: LocalizeFunc = empty;
```

Comment:
```suggestion
  @internalProperty() private _localize: LocalizeFunc = () => "";
```

Reply:
this is being removed?, so change not needed

Reply:
we would still need to init the supervisor with a stub localize function, for when we render before the translations are loaded. We didn't do that before btw :-)